### DISCUSSION TITLE
How to remove the pop panel  on links

### DISCUSSION THREAD

**Discussion by hantsy** [community/community#44957] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In these days, I found when I am moving my cursor to any links in the GitHub issues, it will popup a panel to display more preview info of this link.

But personally, the experience is BAD, when I move out of link, it is never disappeared. 

![Image 001](https://user-images.githubusercontent.com/854520/213846952-ccff2e2b-8d23-471c-81bd-bc65327e5191.png)

I do not like it, how to disable it?

**Comment by radar-blip** [comment]
Its a known bug #44630, the GitHub team is working on a fix.

**Reply by queenofcorgis** [reply]
👋🏼 It looks like we have a fairly promising fix in the works - thread [here](https://github.com/orgs/community/discussions/44630#discussioncomment-4833394) for more information and updates.

**Reply by 2bndy5** [reply]
Yes, we're aware of the thread to address the known bug... 

I think the purpose of this thread is to discuss an option to disable the pop-ups (perhaps in account settings). It has been requested/suggested multiple times in the #44630 discussion. Personally, I would greatly appreciate an option to prevent distracting pop-ups when using GitHub.
